capstone/src/main/webapp/script.js

```diff
@@ -388,21 +388,23 @@ function translatetext(){
 /** 
  * Translates a given text
  */
-function translatetext2(text, language){
+async function translatetext2(text, language){
```

Comment:
Can you delete the translate functions to avoid merge conflicts when I push the updated ones now